common/test/acceptance/pages/lms/fields.py

```diff
@@ -0,0 +1,224 @@
+# -*- coding: utf-8 -*-
+"""
+Mixins for fields.
+"""
+from bok_choy.promise import EmptyPromise
+
+from ...tests.helpers import get_selected_option_text, select_option_by_text
+
+
+class FieldsMixin(object):
+    """
+    Methods for testing fields in pages.
+    """
+
+    def field(self, field_id):
+        """
+        Return field with field_id.
+        """
+        self.wait_for_ajax()
```

Comment:
Same here. self.wait_for_ajax command should never be required at the beginning of a method. 
Instead it should be put at the end of whatever action triggered an ajax call, like where you clicked on something.